packages/components/src/DataTable/Item/DataTableRow.tsx

```diff
@@ -119,6 +121,18 @@ const DataTableRowLayout = forwardRef(
 
 DataTableRowLayout.displayName = 'DataTableRowLayout'
 
+const getRowHoverColor = (
+  theme: DefaultTheme,
```

Comment:
Use `Theme` from `@looker/design-tokens`

```suggestion
  theme: Theme,
```